website/docs/r/ssh_secret_backend_role.html.md

```diff
@@ -62,6 +62,8 @@ The following arguments are supported:
 
 * `allowed_critical_options` - (Optional) Specifies a comma-separated list of critical options that certificates can have when signed.
 
+* `allowed_domains_template` - (Optional) Specifies if `allowed_domains` can be declared using identity template policies. Non-templated domains are also permitted.
```

Comment:
nit: ideally we split lines at max column length 100. I know a lot of lines in this file go over, and we need to update those at some point; but still better to keep to preferred patterns whenever we add new lines/code :)

```suggestion
* `allowed_domains_template` - (Optional) Specifies if `allowed_domains` can be declared using
  identity template policies. Non-templated domains are also permitted.
```